tests/test_bidsArchive.py

```diff
@@ -185,6 +185,25 @@ def testGetImages(bidsArchive4D, sample4DNifti1, bidsArchiveMultipleRuns,
     assert archiveImages != []
     assert len(archiveImages) == 2
 
+    # Test getImages' results are the same with or without using decompression
+    dataset1 = BidsArchive(smallBidsDatasetFactory.get(suffix='1'))
```

Comment:
Wasn't clear to me that smallBidsDatasetFactory is downloading compressed files. Is it that accession number that has compressed data, or all have compressed data?

Reply:
Strictly speaking, it's [only recommended by the BIDS standard](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#imaging-files) -- all OpenNeuro datasets I've seen so far have compressed data though.

I'll add a check to ensure that the downloaded dataset does have compressed data.